docs/resources/samples/macros-power-automate.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
This might belong under "Troubleshooting" instead of "Samples"

Reply:
Tracking with feature 4924924 (task 4989446)